block/block.go

```diff
@@ -194,7 +194,8 @@ func GenesisBlockInit() (*Block, error) {
 	}
 
 	rewardAddress, _ := ToScriptHash(config.InitialIssueAddress)
-	payload := NewCoinbase(EmptyUint160, rewardAddress, Fixed64(config.InitialIssueAmount))
+	donationProgramhash, _ := ToScriptHash(config.DonationAddress)
+	payload := NewCoinbase(donationProgramhash, rewardAddress, Fixed64(config.InitialIssueAmount))
```

Comment:
What's the difference between using empty vs donation address as sender?

Reply:
this is a bug， only the coinbase transaction of genesisblock has a  EmptyUint160 sender.